accounts-db/src/accounts_hash.rs

```diff
@@ -250,6 +253,21 @@ impl HashStats {
             ("collect_snapshots_us", self.collect_snapshots_us, i64),
             ("num_snapshot_storage", self.num_snapshot_storage, i64),
             ("scan_chunks", self.scan_chunks, i64),
+            (
```

Comment:
Maybe these stats already do what you are after?

https://github.com/solana-labs/solana/blob/8c69a0ec389372f91d136a9dcc9dc7610b235463/accounts-db/src/cache_hash_data_stats.rs#L39-L53

Reply:
Yeah. These stats are what I want. I removed the redundant stats that I added.